gammapy/spectrum/fit.py

```diff
@@ -414,40 +406,8 @@ def _make_fit_result(self, parameters):
 
     def est_errors(self):
         """Estimate parameter errors."""
-        if self.method == 'iminuit':
-            self._est_errors_iminuit()
-        else:
-            raise NotImplementedError('{}'.format(self.method))
-
-        for res in self.result:
-            res.covar_axis = self.covar_axis
-            res.covariance = self.covariance
-            res.model.parameters.set_parameter_covariance(self.covariance, self.covar_axis)
-
-    def _est_errors_iminuit(self):
-        # The iminuit covariance is a dict indexed by tuples containing combinations of
-        # parameter names
-
-        # create tuples of combinations
-        parameter_names = list()
-        parameter_names_minuit = self._iminuit_fit.parameters
-        for par, parname in zip(self._model.parameters.parameters,
-                                parameter_names_minuit):
-            if not par.frozen:
-                parameter_names.append(parname)
-
-        self.covar_axis = self._model.parameters.free
-        parameter_combinations = list(product(parameter_names, repeat=2))
-
-        if self._iminuit_fit.covariance:
-            iminuit_covariance = self._iminuit_fit.covariance
-            cov = np.array([iminuit_covariance[c] for c in parameter_combinations])
-        else:
-            # fit did not converge
-            cov = np.repeat(np.nan, len(parameter_combinations))
-
-        cov = cov.reshape(len(parameter_names), -1)
-        self.covariance = cov
+        # TODO: add this back once fitting backends support optimisation
+        # and error estimation as separate steps
```

Comment:
Are you sure this works? I would have expected a ``pass`` here ...

Reply:
```
>>> def spam():
...     """It works if there's a docstring!"""
... 
>>>
```

Reply:
Always write docstrings :point_up: